{"uuid": "1c04bc7b-bbdb-45df-8b5d-11a8ba180a66", "children": ["d7ca2386-61bb-4764-aabc-34a21fe5e3be", "97e46249-6b15-46d2-892c-1837ece62d0e", "6843b84d-7a06-4222-8627-179a7221b112", "c40e691b-09d2-4b79-8668-97cf7025e364", "f22825c1-47f2-4cf9-abfa-017101d24d07", "9088a597-35bc-446b-b504-7a0c8fa07384"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'ifname'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 403, in get_interface_details\n    wifi_info[j[\"ifname\"]] = [j[\"config\"][\"ssid\"], j[\"config\"][\"encryption\"], \"\"]\n"}, "attachments": [{"name": "ucentral_config: ", "source": "6bb7adbb-3536-45fe-9657-2e6169ed8948-attachment.attach"}, {"name": "Time Took to apply Config: 231.75397181510925", "source": "6531a29c-b315-49bb-b27a-e7f3975e09d0-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "e5f93387-dc22-4e5e-867a-8b0213d4d31f-attachment.attach"}, {"name": "AP LOgs: ", "source": "0636be77-1ece-4764-bb6c-cd77aa55eefb-attachment.attach"}], "start": 1629074831359, "stop": 1629075148416}], "start": 1629074831359, "stop": 1629075148878}